packages/terra-slider/src/Slider.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+import styles from './Slider.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+  * Slider label value.
+  */
+  labelName: PropTypes.string.isRequired,
+
+  /**
+  * Minimum value in the range.
+  */
+  minimumValue: PropTypes.number.isRequired,
+
+  /**
+  * Maximum value in the range.
+  */
+  maximumValue: PropTypes.number.isRequired,
+
+  /**
+  * Default or initial value for the slider.
+  */
+  defaultValue: PropTypes.number.isRequired,
+
+  /**
+  * When true, will disable the field.
+  */
+  isDisabled: PropTypes.bool,
+
+  /**
+  * Minimum value label to display on the left side of the slider.
+  */
+  minimumLabel: PropTypes.string,
+
+  /**
+  * Maximum value label to display on the right side of the slider.
+  */
+  maximumLabel: PropTypes.string,
+
+  /**
+  * Function to trigger when user changes the input value. Sends parameter {Event} event.
+  */
+  onChange: PropTypes.func,
+};
+
+const defaultProps = {
+  isDisabled: false,
+};
+
+const Slider = (props) => {
+  const {
+    isDisabled,
+    minimumValue,
+    maximumValue,
+    defaultValue,
+    labelName,
+    minimumLabel,
+    maximumLabel,
+    onChange,
+  } = props;
+
+  const sliderRef = useRef();
+
+  useEffect(() => {
+    sliderRef.current.style.setProperty('--terra-slider-progress-status', `${sliderRef.current.value}%`);
+  }, []);
+
+  function handleValueChange(event) {
+    event.currentTarget.style.setProperty('--terra-slider-progress-status', `${event.currentTarget.value}%`);
+    event.currentTarget.setAttribute('aria-valuenow', `${event.currentTarget.value}`);
+
+    if (onChange) {
+      onChange(event);
+    }
+  }
+  const theme = React.useContext(ThemeContext);
+
+  const sliderClassNames = cx([
+    'slider',
+    theme.className,
+  ]);
+
+  const minLabel = minimumLabel || minimumValue;
+  const maxLabel = maximumLabel || maximumValue;
+
+  return (
+    <div className={sliderClassNames}>
+      <label className={cx('label')} id="sliderLabel">
+        {labelName}
+      </label>
+      <span className={cx('min-label')}>
+        {minLabel}
+      </span>
+      <input className={cx('input-range')} type="range" ref={sliderRef} aria-labelledby="sliderLabel" aria-valuemin={minimumValue} aria-valuemax={maximumValue} aria-valuenow={defaultValue} defaultValue={defaultValue} disabled={isDisabled} min={minimumValue} max={maximumValue} onChange={handleValueChange} />
```

Comment:
You shouldn't need aria-valuemin and aria-valuemax since you are using the native input control.  We shouldn't need aria-valuenow once we update the implementation to allow consumers to pass the value.

Reply:
It was confirmed in a UX/ accessibility meeting that these values need to be present for the slider. I will double check if these are required.

Reply:
We should be able to read the ARIA and APG documentation for this.  I would expect to get this for free for a semantic HTML element.

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/slider_role

HTML's <input type="range"> implicitly has the role of slider. Do not use aria-valuemax or aria-valuemin attributes on <input type="range"> elements; use min and max instead. Otherwise, any global aria-* attributes and any other aria-* attributes applicable to the slider role.

Reply:
@eawww @chrismichalewicz  Can you confirm that these aria props can be removed.

Reply:
I will do a quick change and see if it effects the screenreader behavior after removing them and post here.

Reply:
The documentation states that they should not be there.  We should not need additional confirmation.